README.md

```diff
@@ -1 +1,52 @@
 # with-drivers
+
+[![Build Status][ci-image]][ci-url]
+[![Coverage Status][coverage-image]][coverage-url]
+[![NPM version][npm-image]][npm-url]
+[![Dependencies Status][dependencies-image]][dependencies-url]
+[![DevDependencies Status][devdependencies-image]][devdependencies-url]
+
+Middleware that instantiates and adds drivers to the context inside `driver` property. It accepts an optional config property with keys matching driver names.
+
+```js
+const handler = require('@alpha-lambda/handler');
+const { S3, SQS } = require('@alpha-lambda/aws-drivers');
+const withDrivers = require('@alpha-lambda/with-drivers');
+
+const config = {
+  sqs: { level: 'debug' }
+};
+
+module.exports.handler = handler()
+  .use(withDrivers({
```

Comment:
hmm, this isn't much less code than `x: new X(config)`,

I know we can't spend a whole lot of time on this, what about making the 'with' middleware part of alpha-lambda, and it just assigns to context, plus the drivers exposing the "map-new" helper?

example:

```js
const handler = require('@alpha-lambda\handler')
const { fromConfig } = require('@alpha-lambda\aws-drivers');

const conf = {
  aws: {
    dynamodb: { ... },
    s3: { ... }
  }
};

// not sure of initialize, etc.
const drivers = fromConfig(conf.aws);

module.exports.handler = alpha()
  .withContext({ drivers })
))
```

I would think withContext can do a one-level deep Object.assign without having to get in to fancy _.merge behaviour